### ISSUE TITLE
Calendar: Incorrect duration value after adding/editing event with All Day unchecked

### ISSUE THREAD

**Issue by brianjuan** [infor-design/enterprise#3107]
**Describe the bug**
Calendar: Incorrect duration value after adding/editing event with All Day unchecked.

**To Reproduce**
1. Go to http://4230-beta0-enterprise.demo.design.infor.com/components/calendar/example-index.html#
2. Select November 11, 2019
3. Click + icon at the top right, then click "Add Event(Modal)"
4. In the Request Time Off modal, uncheck All Day checkbox
5. Select dates: from 11/11/2019 to 11/13/2019
6. Select time: from 12:00 AM to 12:00 PM
7. Click Submit button
8. See issue: "2 Days 60 Days" will be displayed in the duration

**Expected behavior**
Duration should display correct value, eg. "3 Days".

**Version**
- ids-enterprise: v4.15.0 to v4.23.0-beta0

**Screenshots**
![image](https://user-images.githubusercontent.com/39508864/68757194-7b0abc80-0646-11ea-8c89-fb5047fab5fb.png)

**Platform**
 - All OS and browsers

**Additional context**
This is also happening when editing an event.

**Comment by tmcconechy** [comment]
Working on this with https://github.com/infor-design/enterprise/issues/3111?email_source=notifications&email_token=AAGGZS2SO2DZ22VSVT3YUUTQTPVAPA5CNFSM4JM2L2NKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4HY7X2GA

**Comment by brianjuan** [comment]
QA Failed. Unable to create multiple days event. Tested in http://4230-rc0-enterprise.demo.design.infor.com/components/calendar/example-index.html# 

Steps:
> * Select November 11, 2019
> * Click + icon at the top right, then click "Add Event(Modal)"
> * In the Request Time Off modal, uncheck All Day checkbox
> * Select dates: from 11/11/2019 to 11/13/2019
> * Select time: from 12:00 AM to 12:00 PM
> * Click Submit button

See issue: Created event is only on November 11
![image](https://user-images.githubusercontent.com/39508864/68920040-0901b700-07af-11ea-8238-efdb1b3093f1.png)

**Comment by brianjuan** [comment]
I noticed that the duration is inconsistent, the value changes after focusing out from the created event. I also noticed that "All Day" is checked when you open the event, I think this affects the changing of duration. See attached video below.

Tested in http://4230-rc1-enterprise.demo.design.infor.com/components/calendar/example-index.html 
![calendar-duration](https://user-images.githubusercontent.com/39508864/69215923-db997c80-0ba5-11ea-93bc-d21909bd919c.gif)
